### DISCUSSION TITLE
Hide text labels on taskbar?

### DISCUSSION THREAD

**Discussion by Jaxel** [valinet/ExplorerPatcher#1384] [Ideas]
Right now, in order to hide text labels, you have to enable "always combine" taskbar icons. I would like to hide text labels, without combining icons.

Taskbar Tweaker was able to do this. I would like to be able to do this with Explorer Patcher, without having to install TT.

**Comment by pyrates999** [comment]
I don't see a problem with installing both explorer patcher and taskbar tweaker.  It shouldn't effect stability of your system.

**Reply by Jaxel** [reply]
Well I prefer to have as few extraneous apps installed as possible.

**Reply by sveinmagnus** [reply]
The problem is that Taskbar Tweaker doesn't work on the newer Windows 11's without adding yet another tool, and is not compatible with the ExplorerPatcher Windows 10-style taskbar.

**Reply by pyrates999** [reply]
No reason to install taskbar tweaker on windows 11 now.  And the taskbar options in EP's reimplemented windows 10 taskbar work with the settings in taskbar settings in the windows 11 settings application for not combining the same application and show text labels.

**Comment by ghost** [comment]
Agree with Jaxel. Please add this feature. Thank you

**Comment by lonbluster** [comment]
Yes, labels make little sense when the bar is vertical

**Comment by tolgabalci** [comment]
I think more often than not, the labels are not useful since you can only see a small portion.

The issue from my perspective is that it is not obvious you have to use "always combine" before the labels automagically disappear.  Can the properties menu at least be improved to make that more apparent?  I had to google it, which lead me here before I could figure out how to get rid of the labels.

FYI, I AM using the vertical orientation, although, even for horizontal I don't think I like the labels.

**Reply by justaninja** [reply]
Thanks, that fixed it! But it would be nice to be able to hide the labels without having to combine taskbar icons. Would appreciate the ability to hide labels as a separate feature still.

**Reply by pyrates999** [reply]
That is something Microsoft would need to add.

**Comment by brunogenovese** [comment]
It is 2024 and using the latest version on Windows 11 I can't seem to find an option either in Properties nor in the Windows taskbar settings where I can set the option to "Always combine".

The closest such configuration is in Windows Personalization>Taskbar >Taskbar behaviors>Combine taskbar buttons and hide labels... which I have set to "Always" but has no effect.  Taskbars still show the labels when ExplorerPatcher is operating.

**Reply by pyrates999** [reply]
The year has nothing to do with what you reported.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Comment by brunogenovese** [comment]
@pyrates999 
Both are updated to the latest versions available

- Windows 11 version 23H2 (OS Build 22631,3007)
- ExplorerPatcher 22621.2861.62.2
- The only settings I think I changed are those to make the icons small.  I remember looking for settings to stop the labels from displaying NEXT to OPEN application icons, but not finding any that worked.  The labels that show when you hover on an icon do work properly and only display while hovering.
- I have no other software installed to modify the UI.

**Reply by pyrates999** [reply]
Try undoing the setting you changed in EP properties application that made the icons small and see if that fixes it.

**Reply by izogfif** [reply]
After installing Windows from scratch (Version 23H2 (OS Build 22631.3447), I was unable to make "never combine, hide labels" to work. I went back to old computer, right-clicked taskbar, selected "Properties" from the pop-up menu, then went to "About" and clicked "Export current settings". 
On a fresh Windows install I right-clicked taskbar, selected "Properties" from the pop-up menu, then went to "About" and clicked "Import settings", then selected the .reg file I got on an old computer. Now it works again. I have no idea why I am unable to achieve the same result by changing settings via UI. here is that .reg file (I had to add .txt to it in order for GitHub to allow file upload).
[ExplorerPatcher_22621.3296.64.1.reg.txt](https://github.com/valinet/ExplorerPatcher/files/15052460/ExplorerPatcher_22621.3296.64.1.reg.txt)

**Reply by olaims** [reply]
> After installing Windows from scratch (Version 23H2 (OS Build 22631.3447), I was unable to make "never combine, hide labels" to work. I went back to old computer, right-clicked taskbar, selected "Properties" from the pop-up menu, then went to "About" and clicked "Export current settings". On a fresh Windows install I right-clicked taskbar, selected "Properties" from the pop-up menu, then went to "About" and clicked "Import settings", then selected the .reg file I got on an old computer. Now it works again. I have no idea why I am unable to achieve the same result by changing settings via UI. here is that .reg file (I had to add .txt to it in order for GitHub to allow file upload). [ExplorerPatcher_22621.3296.64.1.reg.txt](https://github.com/valinet/ExplorerPatcher/files/15052460/ExplorerPatcher_22621.3296.64.1.reg.txt)

crazy, I tried your .reg and it works 🤯🤯🤯🤯 no more labels close to the icons. Thanks!!

I had the "always combine" set up, but it was doing nothing till I used your settings.

**Comment by Beej126** [comment]
This worked on my Win11 23H2: [Windhawk - Disable Grouping](https://windhawk.net/mods/taskbar-grouping)

First put the native Win11 Taskbar "Combine taskbar buttons and hide labels" setting to "Always"
and then use this Windhawk extension to disable grouping.

Just have to say what a silly little gap msft has left us with... from web searching one can readily see many want this option.
